packit/config/package_config.py

```diff
@@ -273,8 +290,35 @@ def parse_loaded_config(
             raw_dict=loaded_config,
             config_file_path=config_file_path,
             repo_name=repo_name,
+            spec_file_path=spec_file_path,
         )
         return package_config
     except Exception as ex:
         logger.error(f"Cannot parse package config. {ex}.")
         raise PackitConfigException(f"Cannot parse package config: {ex}.")
+
+
+def get_local_specfile_path(directories):
+    """
+    Get path of the local spec file if present.
+    :param directories: dirs to find the spec file
+    :return: str path of the spec file
+    """
+    for dir in directories:
+        files = [os.path.basename(path) for path in glob(os.path.join(dir, "*.spec"))]
```

Comment:
function name and docstring say the function returns a **path**, but `os.path.basename(path)` returns just the file name, not path